maths/square_root_using_netwon_repson.py

```diff
@@ -0,0 +1,19 @@
+# Finding Square Roots Using Newton's(Newton–Raphson) Method
+n = input("Enter a number to find square root: ")
```

Comment:
This line causes our automated testing to fail.  See CONTRIBUTING.md.

Please encapsulate your algorithm into a function that could be dropped into a larger program:
`def sqrt_using_newton_raphson(n: float) -> float:`

Please add a few doctests that compare the results with `math.sqrt()`.  Your function should behave just like `math.sqrt()` including using the same exceptions and exceptions messages.